lib/utils/getServerlessConfigFile.js

```diff
@@ -108,23 +63,24 @@ const handleJsOrTsConfigFile = (configFile) =>
     }
   });
 
-const getServerlessConfigFile = memoizee((serverless) =>
-  getServerlessConfigFilePath(serverless).then((configFilePath) => {
+const getServerlessConfigFile = memoizee(
+  async (serverless) => {
+    const configFilePath = serverless.configurationPath;
     if (!configFilePath) return null;
     const fileExtension = path.extname(configFilePath);
     const isJSOrTsConfigFile = fileExtension === '.js' || fileExtension === '.ts';
 
-    return (isJSOrTsConfigFile
-      ? handleJsOrTsConfigFile(configFilePath)
-      : readFile(configFilePath)
-    ).then((config) => {
-      if (_.isPlainObject(config)) return config;
-      throw new ServerlessError(
-        `${path.basename(configFilePath)} must export plain object`,
-        'INVALID_CONFIG_OBJECT_TYPE'
-      );
-    });
-  })
+    const config = isJSOrTsConfigFile
+      ? await handleJsOrTsConfigFile(configFilePath)
+      : await readFile(configFilePath);
+
+    if (_.isPlainObject(config)) return config;
+    throw new ServerlessError(
+      `${path.basename(configFilePath)} must export plain object`,
+      'INVALID_CONFIG_OBJECT_TYPE'
+    );
+  },
+  { promise: true }
 );
 
-module.exports = { getConfigFilePath, getServerlessConfigFile, getServerlessConfigFilePath };
+module.exports = { getServerlessConfigFile };
```

Comment:
Any particular reason why it's not exporting it directly with `module.exports = getServerlessConfigFile` ?

Reply:
Good point. Technically it exported many values, and I just removed some, it didn't bother me it exports just one, as anyway this whole util is scheduled to be removed with next step (In light of that I don't think there's a point in cleaning this up further)

Reply:
Sounds good, I was just curious :+1: